applications/crossbar/doc/tasks.md

```diff
@@ -110,11 +119,15 @@ curl -v -X GET \
 }
 ```
 
+#### Add a new system task
+
 
 #### Add a new task
 
 > PUT /v2/accounts/{ACCOUNT_ID}/tasks
 
+Note: There are tasks that run against system resources, only for use by the superduper admin (like rate uploading), which can omit `/accounts/{ACCOUNT_ID}` from the URI. Leaving the account in the URI should have no impact.
+
```

Comment:
As I have said, `/v2/tasks` makes a lot of sense when uploading the "global" ratedecks (not tied to an account). It will make sense to include `/accounts/{ACCOUNT_ID}` when we enable creating ratedecks specifically for resellers.

Crossbar and blackhole appear to be the only applications that care about superduper admin status of an account, with the exception of kt_rates, which it needs at the moment.

If there is no account id on the URL, then it uses the auth_account_id instead; why is this a problem? If kz_tasks didn't require an account_id and auth_account_id we could send in undefined for account_id to those task modules that don't care about accounts. As-is, this works just fine.